UI/views/courses.html

```diff
@@ -25,13 +25,13 @@ <h3 class="colHeader">{{lang.panelTitle}} <small>{{term}}</small>
             <strong>
               <span ng-if="item.Link">
                 <a target="_blank" href="{{item.Link}}">
-                  <span class="sr-only">Go to site for</span>{{item.Title}}</a>
+                  <span class="sr-only">Go to site for</span>{{item.Title}} {{item.SectionNumber}}</a>
```

Comment:
If there is no SectionNumber do we just end up with a trailing space? (Which would be fine.)